src/amber/router/session/cookie_store.cr

```diff
@@ -0,0 +1,80 @@
+module Amber::Router::Session
+  # This is the default Cookie Store
+  class CookieStore < AbstractStore
+    property secret : String
+    property key : String
+    property expires : Int32
+    property store : Amber::Router::Cookies::Store
+    property session : Hash(String, String)
+
+    def initialize(@store, @key, @expires, @secret)
+      @session = current_session
+      @session[key] = SecureRandom.uuid
+    end
+
+    def id
+      @session[key]
+    end
+
+    def destroy
+      @session.clear
+    end
+
+    def [](key)
```

Comment:
True, this is one detail about rails that I dislike, the inconsistency, I have seen people ran into issues because they think **session[:key]** is the same as **session["key"]** those 2 have different pointers. Same for params, I think we should keep it simple and remove the ambiguity.